### ISSUE TITLE
Conda build fails with numpy 1.12

### ISSUE THREAD

**Issue by bladwig1** [conda-forge/wrf-python-feedstock#2]
When using numpy 1.12, conda build fails to create a package for wrf-python.  The build is successful, but the test environment creation fails with:

...WARNING conda_build.build:create_env(713): failed to create env, retrying.  exception was: The following specifications were found to be in conflict:
  - numpy 1.12*
  - wrf-python 1.0b1 np112py27_1
Use "conda info <package>" to see the dependencies for each package.
updating index in: /Users/ladwig/miniconda2/conda-bld/osx-64
updating index in: /Users/ladwig/miniconda2/conda-bld/noarch
...ERROR conda_build.build:create_env(717): Failed to create env, max retries exceeded.
Traceback (most recent call last):
  File "/Users/ladwig/miniconda2/bin/conda-build", line 6, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "/Users/ladwig/miniconda2/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 321, in main
    execute(sys.argv[1:])
  File "/Users/ladwig/miniconda2/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 312, in execute
    noverify=args.no_verify)
  File "/Users/ladwig/miniconda2/lib/python2.7/site-packages/conda_build/api.py", line 89, in build
    need_source_download=need_source_download, config=config)
  File "/Users/ladwig/miniconda2/lib/python2.7/site-packages/conda_build/build.py", line 1503, in build_tree
    .format(recipe, e.packages) + "\n\n" + extra_help)
AttributeError: 'UnsatisfiableError' object has no attribute 'packages'

This occurs on conda-forge or running conda build locally (which is how the error above was generated).  

The problem appears to be caused by xarray not pinning their numpy version to "x.x", because if I change the wrf-python recipe to use 'numpy' instead of 'numpy x.x', the build succeeds.  However, I'm not sure why this problem doesn't occur with numpy 1.11.

**Comment by bladwig1** [comment]
This also fails for me:

conda create -n foo -c conda-forge numpy=1.12 xarray

UnsatisfiableError: The following specifications were found to be in conflict:
  - numpy 1.12*
Use "conda info <package>" to see the dependencies for each package.

**Comment by ocefpaf** [comment]
That works for me on `Linux`. What platform are you trying that? If there is indeed a missing package it is not `xarray` but one of its dependencies that relies on the `numpy` ABI.

**Comment by bladwig1** [comment]
I'm on OSX.  I think it's actually pandas that is the issue. It looks like the numpy 1.12 pandas builds for linux got uploaded a day ago, but the mac builds are probably stuck in the Travis CI MACtastrophy that wrf-python is stuck in.  They keep posting status updates about it.  I'll close this issue now that I know what's wrong.  Thanks.

**Comment by ocefpaf** [comment]
Let me know if that is solved once `pandas` makes into the channel.

**Comment by bladwig1** [comment]
Pandas for Mac still hasn't made it in to the channel after 3 days.  Not sure, but you might need to relaunch the build due to the Travis CI Mac crash. I submitted a pull request yesterday that got processed before the one I launched during the Travis CI Mac crash a few days ago.  I think Travis CI might have discarded all Mac jobs from the time during the crash.  The pull request seemed to be stuck in limbo.
